packages/gatsby-plugin-stylus/package.json

```diff
@@ -8,7 +8,6 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.0.0-beta.42",
-    "mini-css-extract-plugin": "^0.2.0",
```

Comment:
Why was it removed here?

Reply:
i they are following through with my changes in https://github.com/gatsbyjs/gatsby/pull/4971 where i missed this one 😳

Reply:
haha 👍